src/core/bonded_interactions/angle_harmonic.hpp

```diff
@@ -27,36 +27,58 @@
  */
 
 #include "angle_common.hpp"
-#include "bonded_interaction_data.hpp"
 
 #include <utils/Vector.hpp>
 #include <utils/math/sqr.hpp>
 
 #include <cmath>
 #include <tuple>
 
-/** Set parameters for the angle potential. */
-int angle_harmonic_set_params(int bond_type, double bend, double phi0);
+/** Parameters for three-body angular potential (harmonic). */
+struct AngleHarmonicBondParameters {
```

Comment:
why are these types have a ```Parameters``` suffix if they actually also implement the energy and force kernels?